weave-js/src/components/Button/Button.tsx

```diff
@@ -142,10 +148,22 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
       </Tailwind>
     );
 
+    const [isTooltipOpen, setIsTooltipOpen] = useState(false);
     if (tooltip) {
-      // span is needed so tooltip works on disabled buttons
-      return <Tooltip content={tooltip} trigger={<span>{button}</span>} />;
+      return (
+        <Tooltip.Provider>
```

Comment:
It feels weird to me that provider is at this level. I think this is what determines things like popup delay? Not sure the radix defaults match the guidelines from design.

Reply:
Yeah, the provider should be configured at the root of the application. We'd need to do this for both weave and app. There was [an attempt](https://github.com/wandb/core/pull/20803) at this a while back, but looks like it ended up being abandoned due to CI struggles. If I have spare time in the next few sprints I could try taking another stab at it.